website/docs/apps/filter_app/README.md

```diff
@@ -76,10 +76,31 @@ its operational environment and physical characteristics.
     ![image](https://github.com/farm-ng/amiga-dev-kit/assets/39603677/4ca99130-db62-415f-a82c-3211fa7d9c5c)
     ![image](https://github.com/farm-ng/amiga-dev-kit/assets/39603677/cfb64d74-a70c-42d3-aee5-c304afed2d45)
 
-:::note NOTE
-For precise calibration of your Amiga, all measurements are taken in meters (m)
-and should originate from the central axis of the robot.
-Please follow the indicated measurement guide to maintain consistency and reliability in data collection.
+:::info INFO
+**Amiga Calibration Guide**
+
+To ensure accurate calibration of your Amiga robot, please adhere to the following measurement guidelines:
+
+1. **Unit of Measurement**: All measurements must be taken in meters (m).
+
+2. **Reference Point**: Begin all measurements from the central axis of the robot.
```

Comment:
Suggestion to clarify what central axis means. I.e., center of the robot (in length & width) at ground level.

Reply:
Clarified here 0ef4d0bf